specification/src/chapter1.adoc

```diff
@@ -3,34 +3,78 @@
 
 == Introduction
 
-This specification provides guidelines for building secure RISC-V systems using RISC-V security building blocks. It is aimed at developers of RISC-V technical specifications, as well as at designers of secure RISC-V systems.
-
-A few example use cases are provided, based on commonly used security deployment models.
-These are not intended to be exhaustive but are common enough to represent a wide range of deployments of secure products. They are accompanied by use case specific security
-guidelines which are intended to help readers implement secure products for their specific use cases.
+This specification provides a superset of normative requirements for building
+secure RISC-V systems using RISC-V security building blocks. It is aimed at
+developers of RISC-V technical specifications, as well as at designers of
+secure RISC-V systems. It is intended to be referenced by other documentation
+that will define a specific subset of requirements applicable for a given use
+case, certification regime, platform or profile.
+
+
+A few non-normative example use cases are provided, based on commonly used
+security deployment models.
+These are not intended to be exhaustive but are common enough to represent a
+wide range of deployments of secure products. They are accompanied by an
+appropriate set of use case specific security
+guidelines which are intended to help readers implement secure products for
+their specific use cases.
 
 The examples may be extended over time as required.
 
-RISC-V is currently not intending to create a security certification programme. This specification is provided as non-normative guidance for developing secure RISC-V systems which are certifiable within existing third party security certification programmes. 
+RISC-V is currently not intending to create a security certification programme.
+This specification is provided as non-normative guidance for developing secure
```

Comment:
This paragraph seems to contradict with the first paragraph that states this specification includes a set of normative requirements.